compiler/rustc_ast/src/ast.rs

```diff
@@ -2886,6 +2886,23 @@ pub struct Fn {
     pub body: Option<P<Block>>,
 }
 
+#[derive(Clone, Encodable, Decodable, Debug)]
+/// A static item (`static`).
+///
+/// E.g., `static FOO: i32 = 42;` or `static FOO: &'static str = "bar";`.
+pub struct Static {
```

Comment:
was just doing a cursory glance through the diff since rustfmt code was slightly tweaked, but thought i'd share food for thought around whether it'd be worth using `StaticItem` nomenclature for consistency with `ConstItem`